x-pack/legacy/plugins/siem/public/containers/global_time/index.tsx

```diff
@@ -26,6 +26,7 @@ interface GlobalQuery extends SetQuery {
 export interface GlobalTimeArgs {
   from: number;
   to: number;
+  timezone?: string;
```

Comment:
We should not it here because we are not returning the timezone from this component. We should add it [here](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/public/pages/hosts/hosts.tsx#L54) like that `export type HostsQueryProps = { timezone?: string } & GlobalTimeArgs;`